spectrum-scanner/Makefile

```diff
@@ -0,0 +1,20 @@
+# Set the name of your application:
+APPLICATION = spectrum-scanner
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
```

Comment:
Just copy pasta. So far the only platform which supports the netopt extension is the kw41z.